packages/code-studio/src/log/LogExport.ts

```diff
@@ -6,6 +6,42 @@ import { logHistory } from './LogInit';
 
 const FILENAME_DATE_FORMAT = 'yyyy-MM-dd-HHmmss';
 
+// List of objects to blacklist
+// '' represents the root object
+export const PATH_BLACKLIST: string[][] = [
+  ['', 'api'],
+  ['', 'client'],
+  ['', 'dashboardData', 'defaultLayout', 'connection'],
+  ['', 'layoutStorage'],
+  ['', 'storage'],
```

Comment:
Any way we can define it so we don't have the extra `''` in front? Annoying to have to define that for every item, and it doesn't help for any valid case (blacklisting the root wouldn't make sense, and even if it did it could be `[]` or `['']`).